.github/CHANGELOG.md

```diff
@@ -2,6 +2,25 @@
 
 <h3>New features since last release</h3>
 
+* Adds a device test suite, located at `pennylane/plugins/tests`, which can be used 
+  to run generic tests on core or external devices calling 
+  
+  >>> pytest pennylane/plugins/tests --device default.qubit --shots 1234 --analytic False                                                                                                                                                                                                                                                                    
+  
+  The command line arguments are optional.
+   
+  * If `--device` is not given, the tests are run on the core devices that ship with PennyLane. 
+  
+  * If `--shots` is not given, a default of 50000 is used.
```

Comment:
Would 50000 be a bit too many for default? :thinking: We have a 1000 as the default for the device creation. Or could it make sense to have a separate default number of shots for analytic and non-analytic cases?

Reply:
Oh, this is a good point @antalszava --- Qiskit has a maximum of 8192 shots as well!

Reply:
Cool, let me set 10,000

Reply:
@mariaschuld just double-checking: would it be 10,000 or 1,000 here?